docs/adguard-for-android/solving-problems/logcat.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: How to get Logcat log
+title: How to get system logs
 sidebar_position: 4
 ---
 
 :::info
 
-This article covers AdGuard for Android, a multifunctional ad blocker that protects your device at the system level. To see how it works, [download the AdGuard app](https://adguard.com/download.html?auto=true)
+This article covers AdGuard for Android, a multifunctional ad blocker that protects your device at the system level. To see how it works, first-hand [download the AdGuard app](https://adguard.com/download.html?auto=true)
 
 :::
 
-## General Instruction
-
-For troubleshooting problems with crashes a regular log is almost always not enough. In such cases to identify the origin of the problem we need the system log. Below is the instruction how to collect and get it.
+Sometimes a regular log may not be enough to identify the origin of the problem. In such cases a system log is needed. Below is the instruction how to collect and get it: via Developer options and Logcat. 
 
 ## Capture a bug report from a device
+
 To get a bug report directly from your device, do the following:
 
-1. Be sure you have [Developer Options](https://developer.android.com/studio/run/device.html#developer-device-options) enabled.
+1. Be sure you have [Developer options](https://developer.android.com/studio/run/device.html#developer-device-options) enabled.
 
 2. In **Developer options**, tap **Take bug report**.
 
 ![Bug report *mobile](https://cdn.adtidy.org/public/Adguard/kb/newscreenshots/En/Android3.1/bugreporten.png)
 
-3. Select the type of bug report you want and tap **Report**.
->After a moment you get a notification that the bug report is ready (see figure 2).
+3. Select the type of the bug report you want and tap **Report**. 
+
+> After a moment you get a notification that the bug report is ready (see figure 2).
```

Comment:
::: note

After a moment, you will see a notification that the bug report is ready (see Figure 2).

:::